include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -554,7 +558,7 @@ struct __parallel_copy_if_static_single_group_submitter<_Size, _ElemsPerItem, _W
 template <typename _ExecutionPolicy, typename _InRng, typename _OutRng, typename _UnaryOperation, typename _InitType,
           typename _BinaryOperation, typename _Inclusive>
 auto
-__parallel_transform_scan_single_group(oneapi::dpl::__internal::__device_backend_tag, _ExecutionPolicy&& __exec,
```

Comment:
Same question about why this change is necessary.  `__parallel_transform_scan_static_single_group_submitter` with a forwarded policy should work and `const _ExecutionPolicy&` should bind to forwarded policy type.

Reply:
The same - this code calls some submitter `__parallel_transform_scan_static_single_group_submitter`